tests/integration/test_cos.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Integration tests for Synapse charm."""
```

Comment:
The docstring could be a bit more specific. This also applies to the other modules.

Reply:
See https://github.com/canonical/synapse-operator/pull/194#discussion_r1492440319